### PR TITLE
Add support for http post of url encoded data

### PR DESCRIPTION
**Pull request by stphivos**

### CODE REVIEW

**Review by javiromero — APPROVED** [review]

File: mws/mws.py [hunk 1]
```diff
@@ -305,31 +305,48 @@ def make_request(self, extra_data, method="GET", **kwargs):
 
         request_description = calc_request_description(params)
         signature = self.calc_signature(method, request_description)
-        url = "{domain}{uri}?{description}&Signature={signature}".format(
+        url_base = "{domain}{uri}".format(
             domain=self.domain,
             uri=self.uri,
+        )
```
Comment (skqr) [inline mws/mws.py]: Is this not py3? `f"this {thang}?"`

File: mws/mws.py [hunk 2]
```diff
@@ -305,31 +305,48 @@ def make_request(self, extra_data, method="GET", **kwargs):
...
+        if method == "POST":
+            url = url_base
+            body += url_data
+            headers.update({"Content-Type": "application/x-www-form-urlencoded"})
+        else:
+            url = url_base + "?" + url_data
 
         try:
-            # Some might wonder as to why i don't pass the params dict as the params argument to request.
+            # Some might wonder as to why i don"t pass the params dict as the params argument to request.
```
Comment (skqr) [inline mws/mws.py]: Small typo.

File: mws/mws.py [hunk 3]
```diff
@@ -305,31 +305,48 @@ def make_request(self, extra_data, method="GET", **kwargs):
...
             response.raise_for_status()
             # When retrieving data from the response object,
             # be aware that response.content returns the content in bytes while response.text calls
             # response.content and converts it to unicode.
 
             data = response.content
             # I do not check the headers to decide which content structure to server simply because sometimes
-            # Amazon's MWS API returns XML error responses with "text/plain" as the Content-Type.
-            rootkey = kwargs.get('rootkey', extra_data.get("Action") + "Result")
+            # Amazon"s MWS API returns XML error responses with "text/plain" as the Content-Type.
```
Comment (skqr) [inline mws/mws.py]: Is this your Black plugin doing some shenanigans?

**Review by skqr — APPROVED** [review]
Looks good, left a couple of (very) minor comments.

File: mws/mws.py (hunk 1 above)
Comment (stphivos) [reply mws/mws.py]: Right it is!

File: mws/mws.py (hunk 3 above)
Comment (stphivos) [reply mws/mws.py]: Not using black in this repo, this was from the manual auto-replacing thing!

**Pull request merged**